apps/src/music/blockly/MusicBlocklyWorkspace.ts

```diff
@@ -86,8 +87,12 @@ export default class MusicBlocklyWorkspace {
     }
 
     this.container = container;
-
-    const toolboxBlocks = getToolbox(blockMode, toolbox);
+    const isStartMode = getAppOptionsEditBlocks() === START_SOURCES;
```

Comment:
minor: can we do this check in `MusicView` and pass in `undefined` for toolbox if we're in start mode? My thinking is just that this being a blockly workspace-specific component doesn't need to know the specifics of what "start mode" means.

Reply:
Yep, sounds good!